docs/user/interfaces/endpoint.rst

```diff
@@ -91,3 +91,62 @@ Explain::
 	  }
 	}
 
+Cursor
+======
+
+Description
+-----------
+
+To get paginated response for a query, user needs to provide `fetch_size` parameter as part of normal query. The value of `fetch_size` should be greater than `0`. In absence of `fetch_size`, default value of 1000 is used. A value of `0` will fallback to non-paginated response. This feature is only available over `jdbc` format for now.
```

Comment:
I think this section can be moved to new example 3 in Request Format in protocol doc: https://github.com/opendistro-for-elasticsearch/sql/blob/master/docs/user/interfaces/protocol.rst#request-format.

As for endpoint, I think the new cursor close API can be documented here?

Reply:
I did trying adding it as example 3 in ProtocolIT ,  but I was not getting correct cursor response, also the explain output was showing non-scroll explain with `from` field. So omitted as it would have required extra plumbing.